docs/developer/creating-a-plugin.rst

```diff
@@ -93,166 +121,39 @@ For example, the Relion plugin has the following structure:
-        libPath = cls.getHome('lib') + ":" + cls.getHome('lib64')
-
-        if binPath not in environ['PATH']:
-            environ.update({'PATH': binPath,
-                            'LD_LIBRARY_PATH': libPath,
-                            'SCIPION_MPI_FLAGS': os.environ.get('RELION_MPI_FLAGS', ''),
-                            }, position=pwutils.Environ.BEGIN)
-
-        # Take Scipion CUDA library path
-        cudaLib = environ.getFirst((RELION_CUDA_LIB, 'CUDA_LIB'))
-        environ.addLibrary(cudaLib)
-
-        return environ
-
-
-Implement validateInstallation() (Optional)
-'''''''''''''''''''''''''''''''''''''''''''
-In the plugin class, we can overwrite the validateInstallation function. In the case of Relion this is not overwritten,
-so this plugin will use Scipion's default validate installation. You can check the current implementation in
-``pyworkflow/plugin.py``.
-
-Defining the plugin binaries (Optional)
-'''''''''''''''''''''''''''''''''''''''
-The next step is to add the code responsible for the installation of the binaries.
-We redefine ``defineBinaries`` in our ``RelionPlugin`` class in the **`__init__.py`**.
-Please note how we have added ``default=True`` to the latest relion binaries. - this means that
-this binary will be installed automatically when we get this plugin (unless specified otherwise).
-
-.. code-block:: python
 
-    # this goes inside class RelionPlugin(Plugin):
-    def defineBinaries(self, env):
-        relion_commands = [('./INSTALL.sh -j %d' % env.getProcessors(),
-                                  ['relion_build.log',
-                                   'bin/relion_refine'])]
-
-        env.addPackage('relion', version='1.4',
-                       tar='relion-1.4.tgz',
-                       commands=relion_commands)
-
-        env.addPackage('relion', version='1.4f',
-                       tar='relion-1.4_float.tgz',
-                       commands=relion_commands)
-
-        # Define FFTW3 path variables
-        relion_vars = [('FFTW_LIB', SW_LIB),
-                       ('FFTW_INCLUDE', SW_INC)]
-
-        relion2_commands = [('cmake -DGUI=OFF -DCMAKE_INSTALL_PREFIX=./ .', []),
-                            ('make -j %d' % env.getProcessors(), ['bin/relion_refine'])]
-
-        env.addPackage('relion', version='2.0',
-                       tar='relion-2.0.4.tgz',
-                       commands=relion2_commands,
-                       updateCuda=True,
-                       vars=relion_vars)
-
-        env.addPackage('relion', version='2.1',
-                      tar='relion-2.1.tgz',
-                      commands=relion2_commands,
-                      updateCuda=True,
-                      vars=relion_vars,
-                      default=True)
+Required
+~~~~~~~~
 
+Scipion plugins are usually composed by the following submodules:
 
-You can find more information about how to add a packages or a module `[here] <scipion-installation-system>`_.
+\__init__.py
+^^^^^^^^^^^^
```

Comment:
How many nested levels do we have? I this new one intentional?

Reply:
There are four levels. The deepest ones can be found in section Plugin Overview. This new structure is proposed considering an approach that show all the information contained in the page in a not-overwhelming way: an index in which is very clear which is required and which not transmits the idea of lighter and friendlier doc. Hence, this is the reason of the new level and structure proposed.